docs/release-notes/8.14.asciidoc

```diff
@@ -0,0 +1,59 @@
+[[release-notes-header-8.14.0]]
+== 8.14
+
+[discrete]
+[[release-notes-8.14.0]]
+=== 8.14.0
+
+[discrete]
+[[features-8.14.0]]
+==== New features
+* Introduces Attack discovery, a new feature that uses AI to identify potential attacks and help you quickly triage multiple alerts ({kibana-pull}181818[#181818]).
+* Creates the **Asset criticality** page within the **Manage** menu, which lets you assign asset criticality levels to your assets ({kibana-pull}179891[#179891]).
+* Adds alert suppression for New Terms rules ({kibana-pull}178294[#178294]).
+* Adds alert suppression for EQL rules with non-sequence queries ({kibana-pull}176422[#176422]).
+* Adds a **Setup guide** markdown field to custom rules ({kibana-pull}178131[#178131]).
+
+[discrete]
+[[enhancements-8.14.0]]
+==== Enhancements
+* Removes the "Technical preview" tag for {esql}; it is now generally available ({kibana-pull}180838[#180838]).
+* Allows you to add calculated values to an {esql} rule's highlighted fields ({kibana-pull}177746[#177746]).
+* Connects {esql} functionality in {elastic-sec} to the `general:enableESQL` advanced setting ({kibana-pull}181616[#181616]).
+* Removes the "Technical preview" tag for custom query rule alert suppression; it is now generally available ({kibana-pull}181698[#[181698]).
```

Comment:
Changing the PR number from the backport PR to the original PR and removing an extra bracket. 

```suggestion
* Removes the "Technical preview" tag for custom query rule alert suppression; it is now generally available ({kibana-pull}181279[#181279]).
```